includes/JWTLogin.php

```diff
@@ -70,37 +70,39 @@ public function __construct() {
     public function execute($subpage) {
         // No errors yet
         $error = '';
-
+    
         // First, need to get the WebRequest from the WebContext
         $request = $this->getContext()->getRequest();
-
-        if (!isset($_POST[self::JWT_PARAMETER])) {
-            $this->getOutput()->addWikiMsg('jwtauth-noparam');
-            return;
-        }
-        // Get raw JWT data
-        $jwtDataRaw = $_POST[self::JWT_PARAMETER];
+    
+        // Get JWT data from Authorization header or POST data
+        if (isset($_SERVER['HTTP_AUTHORIZATION'])) {
+            $jwtDataRaw = $_SERVER['HTTP_AUTHORIZATION'];
+        } else if (isset($_POST[self::JWT_PARAMETER])) {
```

Comment:
Should be changed to `elseif` for performance benefit

Reply:
Incorporated